src/Nethermind/Nethermind.Consensus.AuRa/Contracts/BlockGasLimitContract.cs

```diff
@@ -26,11 +26,11 @@
 
 namespace Nethermind.Consensus.AuRa.Contracts
 {
-    public class BlockGasLimitContract : Contract, IActivatedAtBlock
+    public class BlockGasLimitContract : Contract, IActivatedAt
     {
         private static readonly AbiDefinition Definition = new AbiDefinitionParser().Parse<BlockGasLimitContract>();
         private ConstantContract Constant { get; }
-        public long ActivationBlock { get; }
+        public long Activation { get; }
```

Comment:
activation block was much better but I accept the change :)

Reply:
Yeah it was more descripitve, but we have now Activation at block, or at AuraStep or at anything else actually.